docs/server/installation.rst

```diff
@@ -124,10 +79,7 @@ To activate the virtual environment run the :command:`activate` script:
 
   $ source env/bin/activate
```

Comment:
yep the docs didnt use it and i was reluctant to add it.

I think virtualenvwrapper makes more sense in a dev env